.github/workflows/stale.yml

```diff
@@ -1,35 +1,36 @@
-name: "Mark stale PRs & Issues"
+name: Mark stale PRs & Issues
 
 on:
   schedule:
-  # Run this every hour, so we are not spammed with changes at once. Later we could consider changing this to once a day.
-  - cron: "0 * * * *"
+  - cron: 0 0 * * *  # Runs at 00:00 UTC every day.
+
+permissions:
+  issues: write
+  pull-requests: write
 
 jobs:
-  stale-pr:
-    if: github.repository == 'ethereum/eips'
-    runs-on: ubuntu-latest
-    name: "Mark stale PRs"
-    steps:
-    - uses: actions/stale@v3
-      with:
-        repo-token: ${{ secrets.GITHUB_TOKEN }}
-        days-before-stale: 60
-        days-before-close: 7
-        stale-pr-label: 'stale'
-        stale-pr-message: 'There has been no activity on this pull request for two months. It will be closed in a week if no further activity occurs. If you would like to move this EIP forward, please respond to any outstanding feedback or add a comment indicating that you have addressed all required feedback and are ready for a review.'
-        close-pr-message: 'This pull request was closed due to inactivity. If you are still pursuing it, feel free to reopen it and respond to any feedback or request a review in a comment.'
-  stale-issue:
+  stale:
     if: github.repository == 'ethereum/eips'
     runs-on: ubuntu-latest
-    name: "Mark stale issues"
+    name: Mark Stale Issues
     steps:
-    - uses: actions/stale@v3
+    - uses: actions/stale@532554b8a8498a0e006fbcde824b048728c4178f
       with:
+        # General
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-        days-before-stale: 180
-        days-before-close: 14
-        exempt-issue-labels: 'discussions-to'
-        stale-issue-label: 'stale'
-        stale-issue-message: 'There has been no activity on this issue for two months. It will be closed in a week if no further activity occurs. If you would like to move this EIP forward, please respond to any outstanding feedback or add a comment indicating that you have addressed all required feedback and are ready for a review.'
-        close-issue-message: 'This issue was closed due to inactivity. If you are still pursuing it, feel free to reopen it and respond to any feedback or request a review in a comment.'
+        ascending: true # Since we have so many issues, the stale bot finds it hard to keep track. This makes sure that at least the oldest are removed.
+        # Issue config
+        stale-issue-message: There has been no activity on this issue in six months. It will be closed in a week if no further activity occurs. If you are still pursuing it, feel free to respond to any feedback.
```

Comment:
```suggestion
        stale-issue-message: There has been no activity on this issue for two months. It will be closed in a week if no further activity occurs. If you would like to move this EIP forward, please respond to any outstanding feedback or add a comment indicating that you have addressed all required feedback and are ready for a review.
```

Reply:
180/30 = 6 though?

Reply:
Ah, I'm fine with changing that.  I just copied the original text, which appears to be wrong.

Reply:
```suggestion
        stale-issue-message: There has been no activity on this issue for six months. It will be closed in a week if no further activity occurs. If you would like to move this EIP forward, please respond to any outstanding feedback or add a comment indicating that you have addressed all required feedback and are ready for a review.
```